### PR TITLE
[quickfix] acsphotcte

### PR DESCRIPTION
**Pull request by nmiles2718**
- Adds error handling to ensure the supplied MJD date occurs after SM4.
- Fixes typo in documentation where ACS ISR 2012-02 is referenced instead of ACS ISR 2015-05.

### CODE REVIEW

File: acstools/acsphotcte.py [hunk 1]
```diff
@@ -167,6 +167,9 @@ def _check_inputs(self, **inputs):
             it returns the corresponding key. If the iterable inputs are not
             all the same length, it returns `None`.
         """
+        # MJD date of Servicing Mission 4, 2009-05-11
+        # This is the same date used for the CTE Webtool
+        sm4 = 54962.0
```
Comment (pllim) [inline acstools/acsphotcte.py]: Why is this different from the one that Norman defined in `acs_destripe`?

https://github.com/spacetelescope/acstools/blob/4284f9575943ab1b520e2aa83d27ace4d97d1e9d/acstools/acs_destripe.py#L75

File: acstools/acsphotcte.py [hunk 2]
```diff
@@ -210,6 +213,10 @@ def _check_inputs(self, **inputs):
             )
             LOG.error(msg)
             return
+        # Check to see if the date occurs before SM4
+        if inputs['mjd'] < sm4:
+            LOG.error('Observation MJD must occur after SM4, 54962')
```
Comment (pllim) [inline acstools/acsphotcte.py]: Maybe use f-string to insert the SM4 value instead of hardcoding.

File: acstools/acsphotcte.py [hunk 3]
```diff
@@ -37,11 +37,11 @@
 >>> import numpy as np
 >>> from acstools import acsphotcte
```
Comment (pllim) [inline acstools/acsphotcte.py]: Might as well define a variable?
```suggestion
>>> from acstools import acsphotcte
>>> n_sample = 1000
```

File: acstools/acsphotcte.py [hunk 4]
```diff
@@ -37,11 +37,11 @@
...
 >>> ytransfers = 2048 * np.random.random(size=1000)
```
Comment (pllim) [inline acstools/acsphotcte.py]:
```suggestion
>>> ytransfers = 2048 * np.random.random(size=n_sample)
```

File: acstools/acsphotcte.py [hunk 5]
```diff
@@ -37,11 +37,11 @@
...
->>> fluxes = 20000*np.random.random(size=len(ytransfers))
+>>> fluxes = 20000*np.random.random(size=1000)
```
Comment (pllim) [inline acstools/acsphotcte.py]:
```suggestion
>>> fluxes = 20000*np.random.random(size=n_sample)
```

File: acstools/acsphotcte.py [hunk 6]
```diff
@@ -37,11 +37,11 @@
...
 >>> magnitudes = -2.5 * np.log10(fluxes)
 >>> print(magnitudes[:5])
 [-10.75088228  -9.56321561 -10.10571966  -9.01893015 -10.48463087]
->>> local_skys = 80*np.random.random(size=len(ytransfers))
+>>> local_skys = 80*np.random.random(size=1000)
```
Comment (pllim) [inline acstools/acsphotcte.py]:
```suggestion
>>> local_skys = 80*np.random.random(size=n_sample)
```

File: acstools/acsphotcte.py (hunk 1 above)
Comment (nmiles2718) [reply acstools/acsphotcte.py]: Not sure, the value I use corresponds to the launch date for SM4. The value listed in `acs_destripe.py` is 5 days later on May 16 2009. The discrepancy shouldn't be any issue and so I am happy to conform to the date listed in that file.

Comment (pllim) [reply acstools/acsphotcte.py]: Would be nice if ACS Team can agree on a single value. Please see #118.

**Pull request merged**